lib/thumbs/pull_request_worker.rb

```diff
@@ -764,166 +790,242 @@ def render_reviewers_comment_template
   <% end %>
 <% end %>
 </details>
-      EOS
+EOS
     end
 
     def create_reviewers_comment
       comment = render_reviewers_comment_template
       add_comment(comment)
     end
 
-    def load_thumbs_config
-      thumb_file = File.join(@build_dir, ".thumbs.yml")
-      unless File.exist?(thumb_file)
+    def thumbs_config_path
+      File.join(build_dir, ".thumbs.yml")
+    end
+
+    def thumbs_config
+      return @thumbs_config if @thumbs_config
+      unless thumbs_config_present?
         debug_message "\".thumbs.yml\" config file not found"
-        return false
+        return thumbs_config_default
```

Comment:
The return type should be consistent, `thumbs_config` is returning a `Hash` (or `nil`).

I added `thumbs_config_default` to make for easier testing via irb.